Code/Framework/AzQtComponents/AzQtComponents/DPEDebugViewStandalone/main.cpp

```diff
@@ -165,20 +166,11 @@ namespace AZ
 
 namespace DPEDebugView
 {
-    class DPEDebugApplication : public AzToolsFramework::ToolsApplication
+    class DPEDebugApplication
+        : public AzToolsFramework::ToolsApplication
     {
     public:
-        DPEDebugApplication(int* argc = nullptr, char*** argv = nullptr)
-            : AzToolsFramework::ToolsApplication(argc, argv)
-        {
-            AZ::NameDictionary::Create();
-            AZ::AllocatorInstance<AZ::Dom::ValueAllocator>::Create();
```

Comment:
Why don't we need this allocator instance anymore, or was it unused?

Reply:
This allocator gets created by the PropertyEditorSystem, which gets created when the PropertyManagerComponent is created as part of the ToolsApplication instance
The PropertyMangerComponent contains a [PropertyEditorToolsSystem](https://github.com/o3de/o3de/blob/development/Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyManagerComponent.h#L76-L78) instance for the Document Property Editor.
The PropertyEditorToolsSystem creates a [DocumentPropertyEditor::PropertyEditorSystem](https://github.com/o3de/o3de/blob/development/Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/PropertyEditorToolsSystem.h#L36-L37)

The PropertyEditorSystem then creates the allocator in it's [constructor](https://github.com/o3de/o3de/blob/development/Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/PropertyEditorSystem.cpp#L25-L31)